lib/audit_test.pm

```diff
@@ -56,7 +56,17 @@ sub run_testcase {
     # Export MODE
     assert_script_run("export MODE=$audit_test::mode");
 
-    assert_script_run('./run.bash', timeout => $args{timeout});
+    # Test case 'audit-remote-libvirt' does not generate any logs
+    if ($testcase eq 'audit-remote-libvirt') {
+        # Note: the outputs of run.bash can not be saved to "./$file" so saved to "../$file"
```

Comment:
change "so saved to"    -> ", so save to"

Reply:
> change "so saved to" -> ", so save to"

done.